File: desktop/src/de/fatox/meta/desktop/Launcher.java
package de.fatox.meta.desktop;

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;
import de.fatox.meta.EditorMeta;

public class Launcher {
	public static void main(String[] args) {
		Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
		config.setResizable(true);
		config.setTitle("Meta");
		config.setWindowIcon("icon.png");
		EditorMeta editorMeta = new EditorMeta(new WindowPosModifier());
		MetaDesktopLauncher.INSTANCE.init(config, editorMeta);
	}
}

File: editor-desktop/src/main/kotlin/de/fatox/meta/desktop/Launcher.kt
package de.fatox.meta.desktop

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration
import de.fatox.meta.EditorMeta
import de.fatox.meta.desktop.MetaDesktopLauncher.init

fun main() {
	val config = Lwjgl3ApplicationConfiguration()
	config.setResizable(true)
	config.setTitle("Meta")
	config.setWindowIcon("meta-icon.png")
	val editorMeta = EditorMeta(WindowPosModifier())
	init(config, editorMeta)
}
